ANSWERS.md

```diff
@@ -0,0 +1,18 @@
+What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+- PropTypes will check and ensoure the Props passing between the parent and child componenets will be in the right type and shape. It will allow developer to check the console and know right props was passed and avoid future bugs in the program.
+
+Describe a life-cycle event in React?
+
+- example: 
+    ComponentDidMount will link the data from off site server to local program, and trigger a re-render of the component
```

Comment:
Good answer, can you tell me when componentDidMount will run?

Reply:
after the first render